lib/update.php

```diff
@@ -340,6 +361,28 @@ public function vimeography_addon_update_message($plugin_basename, $plugin_data)
   }
 
   /**
+   * Add a reminder to update an expiring key
+   * for the installed Vimeography theme / addon.
+   *
+   */
+  public function vimeography_key_expiring_message() {
+    $today = date("Y-m-d");
+
+
+    if($expiration_date < $today){
+      echo '<div class="notice notice-warning"><p>';
+        printf( __('Heads up! Your %1$s license is expiring soon! For continued support, activate your plugin <a href="http://www.vimeography.com/%2$s">here</a>', 'vimeography'), $this->expiring_addon->product_name, $this->expiring_addon->plugin_name );
```

Comment:
@davekiss - do you have a way to pull the license ID without connecting directly to your website's API? I can pull the ID with `file_get_contents()` but it could definitely slow down their website. If not, perhaps there should be a way to store ID's locally.